src/main/java/com/jfrog/ide/common/tree/DependencyNode.java

```diff
@@ -9,10 +9,10 @@
  * @author yahavi
  */
 public class DependencyNode extends ComparableSeverityTreeNode implements Serializable, SubtitledTreeNode {
-
     private static final long serialVersionUID = 1L;
 
     private GeneralInfo generalInfo;
+    private boolean indirect = false;
```

Comment:
False is the default value of boolean
```suggestion
    private boolean indirect;
```